File: namsor/ruby/src/Anonymize1Example.rb
require "json"
require "namsor_client"

NamsorClient.configure do |config|
    config.api_key["api_key"] = "YOUR_API_KEY"
end

begin
    response = NamsorClient::AdminApi.new.anonymize_1(
        "source", # source
        true, # anonymized
        "token", # token
    )

    p response
rescue NamsorClient::ApiError => e
    puts "Exception when calling AdminApi#anonymize_1: #{e}"
end

File: namsor/php/src/Anonymize1Example.php
<?php

namespace OSEG\NamsorExamples;

require_once __DIR__ . '/../vendor/autoload.php';

use SplFileObject;
use Namsor;

$config = Namsor\Client\Configuration::getDefaultConfiguration();
$config->setApiKey("X-API-KEY", "YOUR_API_KEY");

try {
    $response = (new Namsor\Client\Api\AdminApi(config: $config))->anonymize1(
        source: "source",
        anonymized: true,
        token: "token",
    );

    print_r($response);
} catch (Namsor\Client\ApiException $e) {
    echo "Exception when calling AdminApi#anonymize1: {$e->getMessage()}";
}
